charts/airflow/templates/config/secret-config-envs.yaml

```diff
@@ -61,6 +63,8 @@ data:
   DATABASE_CELERY_CMD: {{ `echo -n "db+postgresql://$(eval $DATABASE_USER_CMD):$(eval $DATABASE_PASSWORD_CMD)@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DB}${DATABASE_PROPERTIES}"` | b64enc | quote }}
   {{- else if eq "mysql" .Values.externalDatabase.type }}
   DATABASE_CELERY_CMD: {{ `echo -n "db+mysql://$(eval $DATABASE_USER_CMD):$(eval $DATABASE_PASSWORD_CMD)@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DB}${DATABASE_PROPERTIES}"` | b64enc | quote }}
+  {{- else if eq "mssql" .Values.externalDatabase.type }}
+  DATABASE_CELERY_CMD: {{ `echo -n "db+mssql://$(eval $DATABASE_USER_CMD):$(eval $DATABASE_PASSWORD_CMD)@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_DB}${DATABASE_PROPERTIES}"` | b64enc | quote }}
```

Comment:
In my deployment I somehow need to add the ODBC driver here as well. Just wanna let you know that, I definitely do not understand the entire code as you guys do ;) :).
`
db+mssql+pyodbc:
`

Reply:
@thesuperzapper I successfully tested the deployment using your line of code `db+mssql` without `+pyodbc` in addition.
I have to add that my instance is running Airflow version 2.2.2.

Reply:
@Jakal1337 Do you have another Microsoft SQL database that you can test this PR [with Airflow 1.10.15](https://github.com/airflow-helm/charts/tree/main/charts/airflow#how-to-use-a-specific-version-of-airflow)? 
(If you do, make sure to set the value `airflow.legacyCommands = true` to enable airflow 1.10 mode in the chart)

I want to verify if this PR will only work with airflow 2.

Reply:
@thesuperzapper multiple Airflow instances can work with one MSSQL DB using separated local SQL users with assigned default Schematas.
Several days ago I tested version 2.2.1 which does not have odbc drivers included in docker image hence it wasn't able to connect.
I will test with Airflow 1.10.15 in the next days.

Reply:
I think its probably best if we just don't support `externalDatabase.type = mssql` when `airflow.legacyCommands = true`. 

Even the latest version of airflow (`2.2.3`)  has significant issues with MsSQL, and support is labeled as ["experimental"](https://github.com/apache/airflow#requirements).

Reply:
@Jakal1337 One last question, you say that your docker image for `2.2.1` did not contain the ODBC drivers, is this is a custom image, or the official `apache/airflow` image?

I thought the official image has has the drivers since 2.2.0 (see commit https://github.com/apache/airflow/commit/ab7acfd7df25cd1a4c739bf99186267356f1eda6).

Reply:
@thesuperzapper sorry for my late reply on this.
I managed to deploy Airflow 1.10.15 using MS-SQL as backend database. For this I added ODBC drivers in our custom docker image (based on official Airflow Image). Tables where created and except some failures on Worker Pod all other Airflow components where online without issues.

You're right in regarding ODBC drivers in official apache/airflow image!